.gitignore

```diff
@@ -17,7 +17,6 @@
 .eslintrc
 .eslintrc.yaml
 .eslintrc.json
-.eslintrc.js
```

Comment:
I guess this gitignore is copied from somewhere and it could undergo a heavy lifting

Reply:
Yeah, I guess it can be cleaned up. I borrow it from some google link at the beginning, because I don't really knew, what type of files generates IntelliJ and what should be included in the repo.

Reply:
Cleaned.